test/unit/AccountInfoMocking.js

```diff
@@ -565,7 +565,7 @@ describe("AccountInfoMocking", function () {
         expect(err).to.be.false;
     });
 
-    it("should re-create a transaction if sign on demand is enabled and a random node is chosen which is not in the current list", async function () {
+    xit("should re-create a transaction if sign on demand is enabled and a random node is chosen which is not in the current list", async function () {
```

Comment:
I think we might need to add an eslint ignore here or otherwise it will say "unexpected skipped mocha test" on some actions.

Reply:
The test needed to be updated first, which is why it wasn't ready for review earlier. It's done now, so you can check it again.